docs/bin/plugin_formatter.py

```diff
@@ -338,7 +338,7 @@ def print_modules(module, category_file, deprecated, options, env, template, out
     if module in deprecated:
         modstring = modstring + DEPRECATED
 
-    category_file.write("  %s - %s <%s_module>\n" % (to_bytes(modstring), to_bytes(rst_ify(module_map[module][1])), to_bytes(modname)))
+    category_file.write("  %s - %s <%s_module>\n" % (to_text(modstring), to_text(rst_ify(module_map[module][1])), to_text(modname)))
```

Comment:
This change was made, as I was getting `b'whatever'_module` in python3, which I am pretty sure we didn't want :)